#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# (c) ACE 

import os

class Config(object):
    # get a token from @BotFather
    pass
    """
    BOT_TOKEN = os.environ.get("BOT_TOKEN", "7316983749:AAEA9-25AnlhJF6pq1_ig9tNXrfgS-rdxAE")
    API_ID = int(os.environ["API_ID", 24431225]
    API_HASH = os.environ["API_HASH", "98dfcbe440b3ea4ce62338d424c32412"]
    AUTH_USERS = "1843979302"6568611832"6852119483"
# jisko bhi is repo ko deploy karne me error aa raha ho to deploy mat krna xD
